scripts/common/repository.js

```diff
@@ -186,7 +188,7 @@ class Repository {
   }
 
   getPackageJson() {
-    return require(this.getPackageJsonPath());
+    return require(this.getPackageJsonPath()); // eslint-disable-line
```

Comment:
@antleblanc the disabling eslint comment has been moved here, using global `eslint-disable-line` as the super-linter action doesn't recognize rule `import/no-dynamic-require`.

(Error is: Definition for rule 'import/no-dynamic-require' was not found)